l10n_es_facturae/README.rst

```diff
@@ -0,0 +1,194 @@
+    :target: https://github.com/OCA/l10n-spain/tree/17.0/l10n_es_facturae
+    :alt: OCA/l10n-spain
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-spain-17-0/l10n-spain-17-0-l10n_es_facturae
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/l10n-spain&target_branch=17.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+En virtud de la Ley 25/2013, de 27 de diciembre, de impulso de la
+factura electrónica y creación del registro contable de facturas en el
+Sector Público, desde el día 15 de enero de 2015 todas las facturas
+remitidas a las administraciones públicas tienen que ser electrónicas.
+Téngase en cuenta, no obstante, que muchas Administraciones Públicas, y
+entre ellas la Administración General del Estado, han hecho uso de la
+potestad de exonerar de esta obligación a las facturas de hasta 5.000
+euros. Estas facturas electrónicas se enviarán a través de los puntos
+generales de entrada de facturas electrónicas de la administración
+correspondiente. Estos puntos generales le permitirán consultar
+electrónicamente el estado de tramitación de su factura, una vez
+remitida. El de la Administración General del Estado se denomina FACe
+(`www.face.gob.es <http://www.face.gob.es>`__). En estas facturas
+electrónicas habrá que indicar la oficina contable, el órgano gestor y
+la unidad tramitadora, para que llegue correctamente a su destino. La
+administración le proporcionará estos datos.
+
+Informacion sobre el formato:
+
+-  `https://www.facturae.gob.es/formato/Versiones/Esquema_castellano_v3_2_x_06_06_2017_unificado.pdf <https://www.facturae.gob.es/formato/Versiones/Esquema_castellano_v3_2_x_06_06_2017_unificado.pdf>`__
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+Este módulo depende del módulo *account_payment_partner* y sus
+dependencias, que se encuentran en
+`https://github.com/OCA/bank-payment <https://github.com/OCA/bank-payment>`__.
+
+Para generar el archivo XML, hace falta el módulo *report_xml* que se
+encuentra en
+`https://github.com/OCA/reporting-engine <https://github.com/OCA/reporting-engine>`__.
+
+En el caso de querer firmar el formato Facturae desde Odoo, debe
+instalarse la última versión de xmlsig mediante el comando ´pip install
+xmlsig´. La versión mínima de la misma debe ser 0.1.2.
+
+Si se instalan los módulos *account_banking_mandate* o
+*partner_firstname*, se utilizará para el XML generado la cuenta
+bancaria de los mandatos o los apellidos respectivamente.
+
+Configuration
+=============
+
+-  Es necesario ir a los modos de pago e indicar su correspondencia con
+   los códigos de Facturae.
+-  La dirección a la que se remite la factura de venta que queremos
```

Comment:
What about changing this point to something like this:

```
La dirección del partner establecido en la factura, a la cual se va a remitir la factura de venta que queremos exportar,
  debe estar marcada como facturae y debe tener cubiertos los datos de
  Oficina contable, Órgano gestor y Unidad tramitadora.
```

If I have understood it correctly, I think it will be clearer for users who are not familiar with the functionality.